src/app/builder/builder.component.html

```diff
@@ -25,6 +25,16 @@
       width="600"
       [src]="previewIframeSrc"></iframe>
   </div>
+  <div *ngIf="playPlaylist" id="preview-playlist-wrap">
```

Comment:
Yup, it should! And it is doing so for me when I try it out -- not for you? (I force-reset the `previewIframeSrc` when playPlaylist switches, here: https://github.com/PRX/play.prx.org/blob/feat/playlists/src/app/builder/builder.component.ts#L66 -- thought that would take care of it)